src/vips/voperation.lua

```diff
@@ -95,9 +84,9 @@ local voperation_mt = {
                 if gtype == gvalue.image_type then
                     value = match_image:imageize(value)
                 elseif gtype == gvalue.array_image_type then
-                    value = map(function(x)
-                        return match_image:imageize(x)
-                    end, value)
+                    for i = 1, #value do
```

Comment:
Getting rid of the higher-order functions is a bit of a shame, but I can see the reasoning.